independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json

```diff
@@ -0,0 +1,55 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "UK Bank Holidays",
+    "description": "A JSON object containing the bank holidays of the four kingdoms of the United Kingdom supplied by H.M. Government",
+    "version": "1.0",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-bh@outlook.com"
```

Comment:
@martynlesbirel Please add `url` property as well

Reply:
@martynlesbirel Please add url property as well

Please let us know if you do not have one.